basxconnect/core/views/person/search_person_view.py

```diff
@@ -66,13 +59,20 @@ def _display_as_list_item(person):
             hg.SPAN(
                 mark_safe(person.personnumber),
                 style="width: 48px; display: inline-block",
+                _class=ITEM_VALUE_CLASS,
+            ),
+            hg.SPAN(
+                person.name,
+                _class=ITEM_LABEL_CLASS,
+                style="dispay:hidden;",
```

Comment:
This should be ```display: none```. You could use ```visibility: hidden``` but in most cases you will want to use the ```display``` property because ```visibility``` takes visual space.